PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,7 @@
+### What to do if the build fails?
+
+Right now contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build do not worry, no actions are needed from your side. Members of the Bitrise tooling team will come and sort it out for you or inform you if any further actions are needed from your side.
```

Comment:
```suggestion
At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for you patiance. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.
For steps added for the first time, the GitHub option **Allow edits from maintainers** option is required (See documentation on how to select it: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
```

Reply:
Nice edit, there is a typo, I will fix it